index.html

```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
+              <li class="list-area">Bootstrap</li>
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box right" id="project-3">
+        <div class="project-content">
+          <div class="project-text">
+            <h1>Profesional Art<br>Printing Data</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+              <li class="list-area">HTML</li>
+              <li class="list-area">Bootstrap</li>
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box" id="project-4">
+        <div class="project-content">
+          <div class="project-text">
+            <h1>Profesional Art<br>Printing Data</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+              <li class="list-area">HTML</li>
+              <li class="list-area">Bootstrap</li>
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box" id="project-5">
+        <div class="project-content">
+          <div class="project-text">
+            <h1>Profesional Art<br>Printing Data</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+              <li class="list-area">HTML</li>
+              <li class="list-area">Bootstrap</li>
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box" id="project-6">
+        <div class="project-content">
+          <div class="project-text">
+            <h1>Profesional Art<br>Printing Data</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+              <li class="list-area">HTML</li>
+              <li class="list-area">Bootstrap</li>
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+    </section>
+    <section class="about_me" id="about">
+      <div class="about_me-container">
+        <div class="about_me-text">
+          <h1>About me</h1>
```

Comment:
- [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.